src/docfx/build/metadata/ContributionProvider.cs

```diff
@@ -249,14 +249,15 @@ private async Task LoadCommits(Docset docset)
                 {
                     var repo = group.Key;
                     var repoPath = repo.Path;
+                    var repoBranch = bilingual && repo.Branch.EndsWith("-sxs") ? repo.Branch.Substring(0, repo.Branch.Length - 4) : null;
```

Comment:
when bilingual is true, and input repo branch not ending with "-sxs", should we log some warning to inform user?

Reply:
u are right, but we should not log here, we should log warnings at the begining of building, let's check it in another PR

---
In reply to: [229918463](https://github.com/dotnet/docfx/pull/3546#discussion_r229918463) [](ancestors = 229918463)